src/Maestro/Maestro.Web/Controllers/AzDevController.cs

```diff
@@ -46,5 +46,20 @@ public async Task<IActionResult> GetBuildStatus(string account, string project,
                         Convert.ToBase64String(Encoding.UTF8.GetBytes(":" + token)));
                 });
         }
+
+        [HttpGet("getFile/{account}/{project}/{repo}/{*filePath}")]
+        public async Task<IActionResult> GetSourceFile(string account, string project, string repo, string filePath)
```

Comment:
This is an authenticated API right? This API is a bit scary because it basically means that the PAT used by Maestro is usable by anyone to get any file in the repo.  Can we at least scope this down so that it only gets you Version.Details.xml?

Reply:
It is authenticated. As we discussed during the meeting, I'll update this since someone with access to the public repo doesn't necessarily have access to the files in the internal repo.

Reply:
![image](https://user-images.githubusercontent.com/44816719/68500169-ccaee380-020f-11ea-8547-2ce279cfdc21.png)

Comments on this API from old PR version. @chcosta @ChadNedzlek

Reply:
@ChadNedzlek I reduced the scope of the call to the specific file. @alexperovich can give more information about why the file request is set up this way instead of using a more direct call - it was something to do with the way the auth works and what the site needs to do to get access.

Based on other feedback, if the logic is going to be moving entirely then this might be moot, but it would be good to have the information anyway.